dcrpm/main.py

```diff
@@ -184,6 +188,7 @@ def main():
         msg = "exception: {}".format(e)
         logging.getLogger("status").error("exception")
         logging.getLogger().error(msg)
+        return 1
```

Comment:
Not a big deal, but we have a `StatusCode` class in dcrpm/util.py with exit code constants

Reply:
Let's do that as a followup, enough changes in here as it is already :)